API/BookSharing/BookSharing.Infrastructure/BookApi/IBookApi.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Threading.Tasks;
+using Refit;
+
+namespace BookSharing.Infrastructure.BookApi
+{
+    //https://openlibrary.org/isbn/9780140328721.json
+    //https://www.googleapis.com/books/v1/volumes?q=9788324626625+terms
+    public interface IBookApi
```

Comment:
to jest odwołanie do konkretnego serwisu, w tym wypadku googla, nie powinno miec generycznej nazwy tylko bardziej specyficzna, Np IGoogleBookApiClient